skatepark/block-templates/search.html

```diff
@@ -12,7 +12,7 @@
 	<!-- wp:query {"query":{"perPage":10,"pages":0,"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true}} -->
 	<!-- wp:post-template -->
 
-		<!-- wp:post-title {"level":3,"isLink":true,"style":{"typography":{"fontStyle":"normal","fontWeight":"800"}},"fontSize":"huge","style":{"spacing":{"margin":{"top":"1em"}}}} /-->
+		<!-- wp:post-title {"level":3,"isLink":true,"style":{"typography":{"fontStyle":"normal","fontWeight":"800"},"fontSize":"var(--wp--custom--font-sizes--normal)","spacing":{"margin":{"top":"1em"}}}} /-->
```

Comment:
mmh, now that you mention it, it should be just huge, it was in fact me searching fontStyle: normal instead of fontSize: normal in this case, plus the markup for the whole block was off, so I'll undo that